addons/mail/static/src/components/autocomplete_input_view/autocomplete_input_view.xml

```diff
@@ -2,7 +2,7 @@
 <templates xml:space="preserve">
 
     <t t-name="mail.AutocompleteInputView" owl="1">
-        <input class="o_AutocompleteInputView form-control" t-attf-class="{{ className }}" t-on-blur="autocompleteInputView._onBlur" t-on-focusin="autocompleteInputView.onFocusin" t-on-keydown="autocompleteInputView._onKeydown" t-att-placeholder="autocompleteInputView.placeholder" t-ref="root"/>
+        <input class="o_AutocompleteInputView form-control" t-attf-class="{{ className }}" t-on-blur="autocompleteInputView._onBlur" t-on-focusin="autocompleteInputView._onFocusin" t-on-keydown="autocompleteInputView._onKeydown" t-att-placeholder="autocompleteInputView.placeholder" t-ref="root"/>
```

Comment:
`onBlur`, `onFocusin` and `onSelect` are all called from components, so not technically private, so no `_`.

Prefixing methods with `_` has not that much value with code relying heavily on modelling with records and fields. So the only acceptable use is when a method strictly "private", that is called internally and technically by the record itself (and only by the record itself), as an implementation detail

Reply:
(same for `_onKeydown`)